Update FHIR Connecter page

**Description:**
Update the FHIR Connector page [1] with unified and valid synapse API configuration for both the example code and source code in the downloadable zip file. Also, in order to make the scenario working, it is required to add the following message builder and formatter to the axis2.xml/deployment.toml file based on the EI version.

For EI-6.x series, add following to the axis2.xml file.
```
<messageBuilder contentType="application/fhir+json" class="org.wso2.carbon.integrator.core.json.JsonStreamBuilder"/>
```
```
<messageFormatter contentType="application/fhir+json" class="org.wso2.carbon.integrator.core.json.JsonStreamFormatter"/>
```
For EI-7.x series, add the following to the deployment.toml file.
```
[[custom_message_builders]]
content_type = "application/fhir+json"
class = "org.wso2.micro.integrator.core.json.JsonStreamBuilder"

[[custom_message_formatters]]
content_type = "application/fhir+json"
class = "org.wso2.micro.integrator.core.json.JsonStreamFormatter"
```

[1] https://ei.docs.wso2.com/en/latest/micro-integrator/references/connectors/fhir-connector/fhir-connector-example/

**Steps to Reproduce**

1. Create an ESB project and a Connector Exporter project as mentioned in the docs.
2. Deploy the sample artifact given in the docs. Or else, use the downloadable the source code.
3. Invoke the deployed sample API using the following CURL command.
```
curl -v POST -d 
   '  {  "resourceType": "Patient",
           "name": [{"family": "Jhone","given": ["Winney","Rodrigo"]}]
      }' "http://localhost:8290/resources/create" -H "Content-Type:application/json" 
```
4. This will throw the following error.
```
[2020-06-29 15:17:12,184] ERROR

{org.apache.synapse.transport.passthru.util.RelayUtils}
- Error while building Passthrough stream org.apache.axiom.om.OMException: com.ctc.wstx.exc.WstxUnexpectedCharException: Unexpected character '{' (code 123) in prolog; expected '<'
at [row,col

{unknown-source}]: [1,1]
at org.apache.axiom.om.impl.builder.StAXOMBuilder.next(StAXOMBuilder.java:296)
at org.apache.axiom.om.impl.llom.OMDocumentImpl.getOMDocumentElement(OMDocumentImpl.java:109)
at org.apache.axiom.om.impl.builder.StAXOMBuilder.getDocumentElement(StAXOMBuilder.java:570)
at org.apache.axiom.om.impl.builder.StAXOMBuilder.getDocumentElement(StAXOMBuilder.java:566)
at org.apache.synapse.transport.passthru.util.DeferredMessageBuilder.getDocument(DeferredMessageBuilder.java:165)
at org.apache.synapse.transport.passthru.util.RelayUtils.buildMessage(RelayUtils.java:168)
at org.apache.synapse.transport.passthru.util.RelayUtils.buildMessage(RelayUtils.java:121)
at org.apache.synapse.mediators.AbstractListMediator.buildMessage(AbstractListMediator.java:152)
at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:95)
at org.apache.synapse.mediators.base.SequenceMediator.mediate(SequenceMediator.java:281)
at org.apache.synapse.core.axis2.Axis2SynapseEnvironment.mediateFromContinuationStateStack(Axis2SynapseEnvironment.java:807)
at org.apache.synapse.core.axis2.Axis2SynapseEnvironment.injectMessage(Axis2SynapseEnvironment.java:305)
at org.apache.synapse.core.axis2.SynapseCallbackReceiver.handleMessage(SynapseCallbackReceiver.java:578)
at org.apache.synapse.core.axis2.SynapseCallbackReceiver.receive(SynapseCallbackReceiver.java:195)
at org.apache.axis2.engine.AxisEngine.receive(AxisEngine.java:180)
at org.apache.synapse.transport.passthru.ClientWorker.run(ClientWorker.java:284)
at org.apache.axis2.transport.base.threads.NativeWorkerPool$1.run(NativeWorkerPool.java:172)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: com.ctc.wstx.exc.WstxUnexpectedCharException: Unexpected character '{' (code 123) in prolog; expected '<'
at [row,col {unknown-source}
]: [1,1]
at com.ctc.wstx.sr.StreamScanner.throwUnexpectedChar(StreamScanner.java:639)
at com.ctc.wstx.sr.BasicStreamReader.nextFromProlog(BasicStreamReader.java:2052)
at com.ctc.wstx.sr.BasicStreamReader.next(BasicStreamReader.java:1134)
at org.apache.axiom.util.stax.wrapper.XMLStreamReaderWrapper.next(XMLStreamReaderWrapper.java:225)
at org.apache.axiom.util.stax.dialect.DisallowDoctypeDeclStreamReaderWrapper.next(DisallowDoctypeDeclStreamReaderWrapper.java:34)
at org.apache.axiom.om.impl.builder.StAXOMBuilder.parserNext(StAXOMBuilder.java:681)
at org.apache.axiom.om.impl.builder.StAXOMBuilder.next(StAXOMBuilder.java:214)
... 19 more
```

**Affected Product Version:**
EI 6.x and 7.x

--- a/en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md
+++ b/en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md
@@ -16,11 +16,11 @@ To know the further information about these operations please refer this link.
 
 > **Note**: If no ID element is provided, or the value is wrong, the server responds with a HTTP 400 error code and provides an operation outcome identifying the issue.
 
-Before you start configuring the FHIR connector, you also need to download WSO2 EI, and we refer to that location as `<PRODUCT_HOME>`.
+Before you start configuring the FHIR connector, you also need WSO2 MI, and we refer to that location as `<PRODUCT_HOME>`.
 
-Specific message builders/formatters configuration needs to be enabled in the product as shown below before starting the WSO2 EI service.
+Before starting the product, you need to enable configurations for specific message builders/formatters.
 
-If you are using **EI7**, you need to enable this property by adding the following to the **<PRODUCT_HOME>/conf/deployment.toml** file. You can further refer to the [Working with Message Builders and Formatters](../../../references/config-catalog/#http-transport) and [Product Configurations](../../micro-integrator/setup/message_builders_formatters/message-builders-and-formatters/) documentations.
+You need to enable transport by adding the following configurations to the **<PRODUCT_HOME>/conf/deployment.toml** file. You can further refer to the [Working with Message Builders and Formatters](../../../../setup/message_builders_formatters/message-builders-and-formatters/) and [Product Configurations](../../../config-catalog/#http-transport) documentations.
 
 ```
 [[custom_message_builders]]
@@ -32,7 +32,7 @@ content_type = "application/fhir+json"
 class = "org.wso2.micro.integrator.core.json.JsonStreamFormatter"
 ```
 
-If you are using **EI 6**, you can enable this property by doing the following Axis2 configurations in the **<PRODUCT_HOME>\repository\conf\axis2\axis2.xml** file.
+If you are using **EI 6**, you can enable this property by doing the following Axis2 configurations in the **<PRODUCT_HOME>/repository/conf/axis2/axis2.xml** file.
 
 **messageFormatters**
 

